app/views/layouts/root.html.haml

```diff
@@ -0,0 +1,45 @@
+= inside_layout :application do
+  = yield
+
+  = content_for :header do
+    .wrapper
+      .govau--logo
+        = image_tag 'logo-crest-2x.png', width: '65'
+      = #render partial: 'feedback_and_menu'
+      .homeintro
+        %h1
+          %span.govau--text
+          %span.visuallyhidden
+            GOV.AU homepage
+        %p
+          Your guide to finding and using Australian Government services.
+      .links-group
+        %h2 Popular links
+        %ul
+          %li
```

Comment:
As per SITES-447 these should be:

Public Holiday dates - /times-and-dates/australian-public-holidays
Lodge your tax return
Working in Australia
Visiting Australia
What can you bring to Australia
Citizenship
School Holiday Dates - /times-and-dates/school-holidays-and-term-dates
Government Jobs